language/transaction-builder/generator/examples/rust/script_fun_demo.rs

```diff
@@ -1,53 +1,10 @@
 // Copyright (c) The Diem Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-use diem_framework::{encode_peer_to_peer_with_metadata_script, encode_peer_to_peer_with_metadata_script_function, ScriptCall, ScriptFunctionCall};
+use diem_framework::{encode_peer_to_peer_with_metadata_script_function, ScriptFunctionCall};
 use diem_types::{AccountAddress, Identifier, StructTag, TypeTag};
 use serde_bytes::ByteBuf as Bytes;
 
-
```

Comment:
Is this part being removed from the original file, or just not copied?

Reply:
Removed from the original example file (which now tests script funs), but copied to the other example file (which tests tx scripts).